tritium-metrics-jvm/src/main/java/com/palantir/tritium/metrics/jvm/JvmMetrics.java

```diff
@@ -216,6 +215,22 @@ protected RatioGauge.Ratio getRatio() {
         });
     }
 
+    private static void registerJvmBufferPools(TaggedMetricRegistry registry) {
+        JvmBuffersMetrics jvmBuffersMetrics = JvmBuffersMetrics.of(registry);
+        for (BufferPoolMXBean pool : ManagementFactory.getPlatformMXBeans(BufferPoolMXBean.class)) {
+            String poolName = pool.getName();
+            if (poolName.equals("direct")) {
```

Comment:
If you want to be particularly defensive against nulls, you could do something like this:
```suggestion
            if ("direct".equals(poolName)) {
```
Or `Objects.equals(poolName, "direct")` which handles both values being null (although that's impossible for a constant string). I wouldn't expect getName to ever return null either, but we've seen crazier things...

Entirely up to you whether to change this :-]

Reply:
Ah good call. I'll change this to the first idea: `"direct".eqauls(poolName)` for the null protection.